gap/Utils.gi

```diff
@@ -168,3 +168,100 @@ function(m, n)
         return QuoInt(m, n) + 1;
     fi;
 end);
+
+
+# Compute the size of Sp(n, q) according to Theorem 1.6.22 in [1]
+InstallGlobalFunction("SizeSp",
+function(n, q)
+    local m, result, powerOfq, i;
+    if IsOddInt(n) then
+        ErrorNoReturn("Dimension <n> must be even but ", n, " was given.");
```

Comment:
This line is not covered, and I actually would suggest to also test functions with invalid inputs to see how they are dealt with. However, this is an ultra premium extra special wish, so not important.